modules/ROOT/pages/npm-projects.adoc

```diff
@@ -33,9 +33,71 @@ The location of the main {productname} script will be: `+node_modules/tinymce/ti
 * Using a webserver route, or
 * Copying the `+tinymce+` directory to a public folder using a build tool such as Gulp or Webpack.
 
+[[install-premium-plugins]]
+== Install premium plugins
+
+The `+tinymce-premium+` NPM package provides all premium plugins, skins, themes, icons, and language packs for commercial TinyMCE customers.
+
+[IMPORTANT]
+====
+The `+tinymce-premium+` package requires a commercial license. Ensure you have a valid license key before using premium plugins. For information on obtaining a license key, see: xref:license-key.adoc[License Key].
+====
+
+=== Prerequisites
+
+Before installing premium plugins, ensure you have:
+
+* The core `+tinymce+` package installed (see above)
+* A valid commercial license key
```

Comment:
A GPL license key also works. Suggest simply removing "commercial" from this bullet.